### PR TITLE
Merge master 2.9.99

### PR DESCRIPTION
**Pull request by ndellingwood**

### CODE REVIEW

**Comment by ndellingwood** [comment]
Master branch updated with kokkos-kernels 2.9.99, this PR merges master back into develop.

**Comment by srajama1** [comment]
@ndellingwood you mean develop into master ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+- BLAS gemm: Dot-based GEMM Cuda optimization for C = betaC + alphaA^TB - [\#490]https://github.com/kokkos/kokkos-kernels/pull/490)
+- Sorting utilities: [\#461](https://github.com/kokkos/kokkos-kernels/pull/461)
+- SGS: Support multiple rhs in SGS efficiently [\#488](https://github.com/kokkos/kokkos-kernels/issues/488)
+- BLAS trsm: Add support and interface for trsm [\#513](https://github.com/kokkos/kokkos-kernels/issues/513)
+- BLAS iamax: Implement iamax [\#87](https://github.com/kokkos/kokkos-kernels/issues/87)
+- BLAS gesv: [\#449](https://github.com/kokkos/kokkos-kernels/issues/449)
+- sptrsv: Add cusparse tpl support for sparse triangular solve, cudagraphs to fallback [\#555](https://github.com/kokkos/kokkos-kernels/pull/555)
+- KokkosGraph: Output colors assigned during graph coloring [\#444](https://github.com/kokkos/kokkos-kernels/issues/444)
+- MatrixReader: Full matrix market support [\#466](https://github.com/kokkos/kokkos-kernels/pull/466)
+
```
Comment (srajama1) [inline CHANGELOG.md]: SpTrsv : Supernodal trisolve is missing

**Comment by crtrott** [comment]
No this is because develop and release candidate had diverged.

**Comment by srajama1** [comment]
Oh, got it ! We update master after this ?

**Comment by crtrott** [comment]
Master is already updated

**Pull request merged**